server/rest/src/main/java/org/infinispan/rest/resources/SecurityResource.java

```diff
@@ -80,9 +82,18 @@ public Invocations getInvocations() {
             .invocation().methods(PUT).path("/v2/security/roles/{principal}").withAction("deny")
             .permission(AuthorizationPermission.ADMIN).name("ROLES DENY").auditContext(AuditContext.SERVER)
             .handleWith(this::deny)
+            .invocation().methods(POST).path("/v2/security/cache").withAction("flush")
+            .permission(AuthorizationPermission.ADMIN).name("ACL CACHE FLUSH").auditContext(AuditContext.SERVER)
+            .handleWith(this::aclCacheFlush)
             .create();
    }
 
+   private CompletionStage<RestResponse> aclCacheFlush(RestRequest request) {
+      EmbeddedCacheManager cm = invocationHelper.getRestCacheManager().getInstance();
+      return SecurityActions.getGlobalComponentRegistry(cm).getComponent(GlobalSecurityManager.class).flushGlobalACLCache()
```

Comment:
You can cache the ```GlobalSecurityManager``` instead of looking it up on every request I think

Reply:
Will do

Reply:
Rethinking about it: this is such a rare operation, that caching the GSM would be wasteful